tools/scanner/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM mlcommons/mlperf_mobile_flutter
+
+RUN apt-get update >/dev/null && DEBIAN_FRONTEND=noninteractive apt-get install -y unzip
+
+ARG SONAR_SCANNER_DIR=$HOME/.sonar
+
+# Install sonar-scanner
```

Comment:
Why do we need a custom image for this?
Aren't there some ready-to-use solutions?

For example, I was able to quickly find the following github action:
https://github.com/marketplace/actions/sonarqube-scanner

Reply:
> For example, I was able to quickly find the following github action:
https://github.com/marketplace/actions/sonarqube-scanner

With only 2 star and last updated on 8 Jun 2020, I prefer not to use it.

BTW, here is the official one: https://github.com/SonarSource/sonarcloud-github-action
It does not work for compiled languages, though.